Dockerfile

```diff
@@ -18,6 +18,16 @@ RUN cd /workspace \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/* /workspace/ /root/.m2/ \
     && echo "$MAXWELL_VERSION" > /REVISION
 
+# Build clean image with non-root priveledge
+FROM openjdk:11-jdk-slim
+
+COPY --from=builder /app /app
+
 WORKDIR /app
 
+RUN chown 1000:1000 /app && \
+    KAFKA_VERSION=$KAFKA_VERSION && MAXWELL_VERSION=$MAXWELL_VERSION && echo "$MAXWELL_VERSION" > /REVISION
```

Comment:
why are we setting the env variables here?

Reply:
I overlooked that they are no longer used, so i will remove it.

Reply:
I removed it